dev/com.ibm.ws.install.packaging_fat/fat/src/com/ibm/ws/install/packaging/fat/InstallUtilityToolTest.java

```diff
@@ -32,6 +32,7 @@
     public static boolean connectedToRepo = true;
```

Comment:
We should rename InstallUtilityToolTest.java

Reply:
will use 11284 to rename this